### ISSUE TITLE
Issue with parenthesis and bool condition

### ISSUE THREAD

**Issue by FranDepascuali** [nicklockwood/SwiftFormat#192]
I had this first:
```swift
if (self.modules.filter { $0.visibility != .always }.count > 0) {
...
}
```
and after running `SwiftFormat`, it converted it to this:
```swift
if modules.filter { $0.visibility != .always }.count > 0 {
...
}
```
Xcode complains in that line because of missing parenthesis:
<img width="497" alt="screen shot 2017-11-19 at 00 31 01" src="https://user-images.githubusercontent.com/12101394/32985723-01079b1c-ccc1-11e7-8663-51d7c3bd6713.png">

I actually don't understand why is it an error without parenthesis.

Apart from that, everything worked great 👍, thanks!

**Comment by nicklockwood** [comment]
Thanks for the report, I’ll get that fixed asap!

**Comment by nicklockwood** [comment]
@FranDepascuali fixed in 0.30.2

**Comment by FranDepascuali** [comment]
Cool, thank you very much!
